logfilter/logfilter.go

```diff
@@ -30,19 +30,33 @@ var (
 		"_SELINUX_CONTEXT",
 		"__REALTIME_TIMESTAMP",
 		"_PID",
+		"CONTAINER_ID",
+		"CONTAINER_ID_FULL",
+		"CONTAINER_NAME",
+		"CONTAINER_TAG",
+		"MACHINE_ID",
+		"_SOURCE_REALTIME_TIMESTAMP",
+		"_SYSTEMD_INVOCATION_ID",
 	}
 
 	blacklistedUnits = map[string]bool{
-		"splunk-forwarder.service":   true,
+		"splunk-forwarder.service": true,
 		// "docker.service":             true,
 		"diamond.service":            true,
 		"logstash-forwarder.service": true,
 		"kubelet.service":            true,
 		"flanneld.service":           true,
 	}
 
+	blacklistedServices = map[string]bool{
+		"main":                           true,
+		"cluster-autoscaler":             true,
+		"kube-resources-autosave-pusher": true,
+		"kube-resources-autosave-dumper": true,
+	}
+
 	blacklistedStrings = []string{
-		"transaction_id=SYNTHETIC-REQ",
```

Comment:
Is there any harm in keeping both variants? I wouldn't be certain all services have JSON log formatting.

Reply:
fair point, restored